server.js

```diff
@@ -19,11 +12,88 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+// Defines the routes
+app.get("/", (req, response) => {
+const bookGuide = {
+  Routes: [
+    {
+      '/books': 'Get all books',
+      '/books/:bookID': 'Get specific book by ID',
+      '/isbn/:isbn': 'Get a book`s isbn',
+      '/languages/language_code': 'Get book by language',
+      '/authors/:authors': 'Get book by author',
+      '/best': 'Get books sorted by rating, highest to lowest'
+    },
+  ],
+};
```

Comment:
I love this! This isn't a requirement but I love that you found a solution to the problem that the user can't see what routes there is to chose from. Fantastique!